parla/task_runtime.py

```diff
@@ -414,7 +433,7 @@ def _complete_dependency(self):
 
     def _set_state(self, new_state: TaskState):
         # old_state = self._state
-        logger.info("Task %r: %r -> %r", self, self._state, new_state)
+        logger.info("[Task] %r: %r -> %r", self._name, self._state, new_state)
```

Comment:
I noticed that generally str(self.taskid) is more useful than self._name. But it's fine for now.

Reply:
Seconding this too. I don't think the names are unique generally.

Reply:
Either is better than just dumping the whole object like before though